src/providers/twitch/eventsub/Controller.cpp

```diff
@@ -154,133 +222,157 @@ SubscriptionHandle Controller::subscribe(const SubscriptionRequest &request)
 
 void Controller::subscribe(const SubscriptionRequest &request, bool isRetry)
 {
-    qCInfo(LOG) << "Subscribe request for" << request.subscriptionType;
-    boost::asio::post(this->ioContext, [this, request, isRetry] {
-        // 1. Flush dead connections (maybe this should not be done here)
-        // TODO: implement
+    // boost::asio::post(this->ioContext, [this, request, isRetry] {
```

Comment:
Leftover?

Reply:
Fixed in 786cc8b09ef0afcd6ba51bba442381f68f1541e0